pkg/types/v2/file_existence.go

```diff
@@ -26,11 +27,15 @@ import (
 	"github.com/GoogleContainerTools/container-structure-test/pkg/utils"
 )
 
+var defaultOwnership = -1
+
 type FileExistenceTest struct {
 	Name        string `yaml:"name"`        // name of test
 	Path        string `yaml:"path"`        // file to check existence of
 	ShouldExist bool   `yaml:"shouldExist"` // whether or not the file should exist
 	Permissions string `yaml:"permissions"` // expected Unix permission string of the file, e.g. drwxrwxrwx
+	Uid         int    `yaml:"uid"`         // ID of the owner of the file
```

Comment:
does specifying omitempty not work?

Reply:
Yes, I want both fields to be optional like `permissions` because I don't want to break existing tests of users.
As far as I understand, `omitempty` is only relevant for encoding. So even when specifying `omitempty`, both fields will have the value `0` if they are not set in YAML. But `0` is a valid ID for a user or a group.

I think it's up to you to decide what's better here. I'm fine with either way, optional or no optional fields.